public/pages/AnomalyCharts/containers/AnomaliesChart.tsx

```diff
@@ -253,14 +256,40 @@ export const AnomaliesChart = React.memo((props: AnomaliesChartProps) => {
                         heatmapDisplayOption={props.heatmapDisplayOption}
                         isNotSample={props.isNotSample}
                       />,
-                      props.showAlerts !== true
+                      props.isNotSample !== true
```

Comment:
Agreed that it's not best practice. The reason I did `isNotSample` instead of something like `isSample` is to match up with existing code that was using `props.showAlerts`, and to minimize the number of code changes & places where the logic would have to be reversed. There is 62 references of this variable currently.

I would still prefer to leave as-is, but let me know if you feel strongly that this should be switched.

Reply:
understand that would cause a lot of code changes.  Agree to leave as-is.